docs/tutorials/openfoam/spack-openfoam.yaml

```diff
@@ -14,11 +14,11 @@
 
 ---
 
-blueprint_name: spack-openfoam
+blueprint_name: spack-openfoam-v6
```

Comment:
I'm not sure if we need spell `v6` out here.

Reply:
Removed.